fangraphs/scores.py

```diff
@@ -515,3 +515,213 @@ def export(self):
         table = self.page.query_selector(".table-scroll > table")
         dataframe = self._scrape_table(table)
         return dataframe
+
+
+class BoxScore(FilterWidgets):
+    """
+    Scrapes the FanGraphs `Box Score`_ pages.
+
+    .. _Box Score: https://fangraphs.com/boxscore.aspx
+    """
+    _widget_class = scores_.BoxScore
+
+    address = "https://fangraphs.com/boxscore.aspx"
+
+    def __init__(self, **kwargs):
+        FilterWidgets.__init__(self, **kwargs)
+
+        self._tables = None
+
+        self.line_score = None
+        self.play_by_play = None
+        self.data_tables = None
+
+    @property
+    def _tables(self):
+        """
+
+        :return:
+        :rtype: list[bs4.Tag]
+        """
```

Comment:
Replace `:rtype:` docstring directive with function annotation (`->`). Type should be `bs4.ResultSet` instead of `list[bs4.Tag]`.